components/web/php/php-7_0/Makefile

```diff
@@ -0,0 +1,183 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2013, EveryCity Ltd. All rights reserved.
+#
+include ../../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		php
+COMPONENT_VERSION=	7.0.5
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH=	sha256:c41f1a03c24119c0dd9b741cdb67880486e64349fc33527767f6dc28d3803abb
+COMPONENT_ARCHIVE_URL=	http://uk1.php.net/get/$(COMPONENT_ARCHIVE)/from/this/mirror
+
+include $(WS_TOP)/make-rules/prep.mk
+include $(WS_TOP)/make-rules/configure.mk
+include $(WS_TOP)/make-rules/ips.mk
+
+PATH= /usr/bin:/usr/gnu/bin:/usr/sbin
+
+# Apache Paths
+AP_PREFIX=/usr/apache2/2.2
+AP_SYSCONFDIR=/etc/apache2/2.2
+
+# PHP Paths
+PHP_PREFIX=/usr/php/7.0
+PHP_SYSCONFDIR=/etc/php/7.0
+PHP_DATADIR=/var/php/7.0
+
+CFLAGS = $(CC_BITS) -D_POSIX_PTHREAD_SEMANTICS
+CPPFLAGS += -I/usr/include/openldap
+
+COMPONENT_PREP_ACTION =        ( cd $(@D) && autoconf )
+# Configure Options
```

Comment:
Can you make this consistent with the rest of file (i.e. remove empty line after and add empty line before)